### PR TITLE
Fix: comment handling for database in dbc format

### PR DESCRIPTION
**Pull request by malneni**
Updated dbc.py to handle comment for Bus.
For comment Sequence after CM_ is choice. ex: CM_ "Special Release";

Test:
Added new file bus_comment. added funtion "test_bus_comment" in test_database to read commnet & check.

### CODE REVIEW

**Comment by malneni** [comment]
for previous review comments look into closed pullrequest: https://github.com/eerimoq/cantools/pull/210

File: cantools/database/can/formats/dbc.py [hunk 1]
```diff
@@ -1471,7 +1477,12 @@ def _load_bus(attributes):
     except KeyError:
         bus_baudrate = None
 
-    return Bus(bus_name, baudrate=bus_baudrate)
+    try:
+        bus_comment = comments['database']['bus']
+    except KeyError:
+        bus_comment = None
+
+    return Bus(bus_name, baudrate=bus_baudrate, comment = bus_comment)
```
Comment (eerimoq) [inline cantools/database/can/formats/dbc.py:1485]: Remove spaces. should be `comment=bus_comment`.

File: tests/test_database.py [hunk 2]
```diff
@@ -4954,7 +4954,12 @@ def test_floating_point_limits(self):
         # outside of high bound for Delta
         with self.assertRaises(cantools.database.EncodeError):
             message.encode({ **data_max, 'Delta': data_max['Delta']+Decimal('0.001') })
-
+    def test_bus_comment(self):
+        filename = 'tests/files/dbc/bus_comment.dbc'
+        db1 = cantools.database.load_file(filename)
+        self.assertEqual(db1.buses[0].comment, 'SpecialRelease')
+        db2 = cantools.database.load_string( db1.as_dbc_string() )
+        self.assertEqual( db2.buses[0].comment, 'SpecialRelease' )
```
Comment (eerimoq) [inline tests/test_database.py:4962]: Remove line 4961 and 4962. Line 4959 and 4960 are enough.

Also, test dump functionality. Right now only load functionality is tested.

File: cantools/database/can/formats/dbc.py [hunk 3]
```diff
@@ -854,7 +860,7 @@ def _load_comments(tokens):
 
     for comment in tokens.get('CM_', []):
         if not isinstance(comment[1], list):
-            continue
+            comments['database']['bus'] = comment[1]
```
Comment (eerimoq) [inline cantools/database/can/formats/dbc.py:864]: There should either be and else branch or continue here.

File: cantools/database/can/formats/dbc.py (hunk 1 above)
Comment (eerimoq) [reply cantools/database/can/formats/dbc.py:1485]: Fix the previous comment please.

**Pull request merged**